### DISCUSSION TITLE
Erpic — система управления фабрикой мебели

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#7] [Сделанные проекты]
Система учета [Erpic](https://erpic.ru) для фабрики мягкой мебели. Использует [Totum](https://totum.online/) и, в связи с ростом компании, пережила несколько функциональных обновлений.

Сейчас в едином пространстве работает 45 сотрудников с различными уровнями доступа:

- работники цехов
- начальники подразделений 
- снабженец и бухгалтер
- менеджеры по продажам
- руководитель

### Текущие возможности

- создание и учет движения заказов 
- расчет потребности в материалах с учетом резервов и текущих остатков на складе
- расчет плановой и фактической себестоимости готовой продукции
- учет этапов производства:
    - последовательное списание материалов и оперативный контроль остатков
    - выбор исполнителей и начисление им сдельной зарплаты
    - учет незавершенного производства
- взаиморасчеты с контрагентами по банковским счетам и кассе
- расчет и выплата зарплаты
- печать документов: счета, УПД, акты-сверок и др.
- отчетность: продажи, управленческий баланс, отчет о прибылях и убытках с аналитикой по каждой статье, ОСВ для бухгалтера и др.

###  Особенности реализации

- автоматическая загрузка выписок из банка с помощью remotes
- отдельный интерфейс для оптимизации загрузки раскройного станка через префильтры
- уведомления об отрицательных остатках материалов 
- дополнительные валидации, исключающие ошибки сотрудников при вводе данных
- вся управленческая отчетность с графиками в реальном времени

[Скриншоты с комментариями в Miro](https://miro.com/app/board/o9J_lZK-xyY=/?invite_link_id=69525762957)

**Comment by ttm-alexeypolunin** [comment]
**Спасибо за отзыв!**

**Вопрос:** относительно получений данных из банка, может вы делали исследование у кого из банков API нынче позволяет получать информацию?

**Reply by erpic-ru** [reply]
API в разном виде присутствует у всех банков. Но в нашем случае у Модульбанка еще есть вебхуки, что сильно упростило реализацию. Т.е. при зачислении/списании Модульбанк шлет вебхук в remotes со всеми данными транзакции. Totum разбирает платеж: подбирает контрагента, статью ДДС и обновляет остатки по счетам. В общем, получился оперативный обмен без сложной синхронизации.
